index.html

```diff
@@ -4012,6 +4023,16 @@ <h4>canonicalId</h4>
                   </p>
                 </div>
 
+                <div class="issue" title="Observance of canonicalId recommendations">
+                  <p>
+If a resolving party does not use the <code><a>canonicalId</a></code> value
+as its primary ID value for the DID subject and treat all other equivalent
+values as secondary aliases, there may be negative or unexpected issues that 
```

Comment:
```suggestion
values as secondary aliases, there might be negative or unexpected issues that 
```

Reply:
@brentzundel I think your change got applied when I merged some other stuff... if it didn't, we'll fix it.

Reply:
Just checked, it did.